labman/db/support_files/populate_test_db.sql

```diff
@@ -1023,11 +1024,28 @@ BEGIN
             IF idx_row_well <= 6 THEN
                 -- Get information for a sample
                 plating_sample_comp_type_id := sample_type_id;
+
+                sample_offset := idx_row_well - 1;
+                -- select one sample for each plate that is unique to that plate
+                IF idx_row_well = 6 THEN
```

Comment:
@antgonza, it's `idx_col_well = 12`, does that make sense?

Reply:
duh! got it ...

Reply:
> which are the unique samples?

Position F12 in each of the 4 sample plates is the unique sample; in order from Test plate 1 to Test plate 4, the unique samples are
`--"1.SKB8.640193"`
`--"1.SKD1.640179"`
`--"1.SKD5.640186"`
`--"1.SKM6.640187"`

Reply:
Thanks but if you search for 1.SKB8.640193 in the code it appears a few times in the tests, which it makes it confusing to be new ... but I guess is fine ...

Reply:
Correct :) Aside from its usages in the tests for picklist generation, etc, related to the new code I added, 1.SKB8.640193 occurs in:
1) TestStudy.test_samples because it is one of the 27 samples in the study
2) TestStudyHandlers.test_get_study_samples_handler because it is one of the first 20 samples in the study
3) TestGDNAPlateCompressionProcess.test_create, where it is added to a new plate that is created in that test but then destroyed again during the database reset in the tear-down for that test class.

Given this, no need to worry about it being duplicated in any of the other plates that persist in the test database.